src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -153,10 +153,6 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 {{- end }}
 
-{{- define "confluence.sysprop.disableHomeLogAppender" -}}
--DConfluenceHomeLogAppender.disabled=true
-{{- end }}
```

Comment:
now that fluentd is reading the confluence log files directly, we don't need this sysprop any more.